src/coreclr/nativeaot/Runtime/thread.cpp

```diff
@@ -786,6 +790,20 @@ void Thread::HijackReturnAddress(NATIVE_CONTEXT* pSuspendCtx, HijackFunc* pfnHij
     HijackReturnAddressWorker(&frameIterator, pfnHijackFunction);
 }
 
+#if defined(TARGET_ARM64) && defined(__GNUC__)
+static inline void* PacSignPtr(void* ptr)
```

Comment:
We do not need this method because you have already declared this at the top?

Reply:
Done